src/service/employeeService.js

```
@@ -1,6 +1,17 @@
 import axiosInstance from "./axiosInstance";
 
 class EmployeeService {
+  static async register(formData) {
+    try {
+      const response = await axiosInstance.post(`/employee`, {
+        ...formData,
+      });
+      return response.data;
+    } catch (error) {
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)